saas/models/saas_operator.py

```diff
@@ -78,29 +78,15 @@ def build_execute_kw(self, build, model, method, args=None, kwargs=None):
         if self.type == 'local':
             return execute(build.name, SUPERUSER_ID, model, method, *args, **kwargs)
 
-    @staticmethod
-    def _make_action(code):
-        return {
+    @job
+    def build_post_init(self, build, post_init_actions, key_value_dict):
+        key_value_dict.update(self._get_auth_urls(build))
+        post_init_actions += MANDATORY_BUILD_POST_INIT
```

Comment:
Let's make it via method that returns code. It allows easily extend mandatory code